core-bundle/contao/templates/twig/layout/default.html.twig

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+{% set html_attributes = attrs()
+    .set('lang', language|default)
+    .set('dir', 'rtl', rtl) %}
+<html{{ html_attributes }}>
+    <head>
+        {% block head %}
+            <meta charset="UTF-8">
+            <title>{{ head.title }}</title>
+
+            {% block meta %}
+                <meta name="robots" content="{{ head.metaRobots }}">
+                <meta name="description" content="{{ head.metaDescription|u.slice(0, 320) }}">
```

Comment:
```suggestion
                <meta name="description" content="{{ head.metaDescription|u.truncate(320, '…', false) }}">
```

a) `truncate(x)` is the same as `slice(0, x)`
b) Adding an ellipsis after the text if it's too long sounds reasonable to me
c) pass `false` as third param to prevent cutting the last word